### DISCUSSION TITLE
Implement startAfter() and endBefore() methods for Realtime Database

### DISCUSSION THREAD

**Discussion by Zohenn** [firebase/flutterfire#6572] [Feature request] [answered]
As the PR for the new ```get()``` method for Realtime Database (#6208) was already merged, it would be nice to see the ```startAfter()``` and ```endBefore()``` methods implemented as well.

**Comment by Salakar** [comment, marked as answer]
This is planned.

cc @lesnitsky, @Ehesp, @russellwheatley - one to add for DB rework.
